src/Map.js

```diff
@@ -64,8 +69,21 @@ const Map = function({ sidebarOpen, setSidebar, position, setPosition }) {
       properties: event.features[0].properties,
     })
 
+    // using global variables? 🤷
```

Comment:
why not useRef ?

https://medium.com/better-programming/8-awesome-react-hooks-2cb31aed4f3d

Reply:
let's merge this for now. turns out we don't really need `window`, but it still is not a great solution. idk how useRef can help here. let's try to improve this later. 🤷‍♂️